client/Dockerfile

```diff
@@ -4,12 +4,28 @@ WORKDIR /usr/src/app/client
 
 RUN apk add aws-cli git
 
+RUN npm -g config set user root
 RUN npm install --global gulp-cli
 COPY --from=golang:1.16-alpine /usr/local/go/ /usr/local/go/
+
+RUN npm install --location=global @sentry/cli
+ENV SENTRYCLI_SKIP_DOWNLOAD=1
+# RUN mkdir -p /usr/src/app/client
+# RUN chown -R node:node /usr/src/app/client
+# USER node
+
+
+# RUN mkdir ~/.npm-global
+# RUN npm config set prefix ‘~/.npm-global’
+#ENV PATH="~/.npm-global/bin:$PATH"
 ENV PATH="/usr/local/go/bin:${PATH}"
+
+# RUN chown -R root /root/.npm
+
 COPY package.json .
 COPY package-lock.json .
-RUN npm install
+ENV SENTRYCLI_USE_LOCAL=1
+RUN SENTRYCLI_USE_LOCAL=1 npm install --location=global --unsafe-perm
```

Comment:
CMIIW, I think the `SENTRYCLI_USE_LOCAL=1` part can be removed because there's an `ENV` command above

Reply:
And maybe we should use `ARG` instead of `ENV` since `SENTRYCLI_SKIP_DOWNLOAD` and `SENTRYCLI_USE_LOCAL` only needs to be set during build and not the final image

Reply:
Thanks for the great suggestions. I didn't know about `ARG`.